roles/satellite-clone/tasks/main.yml

```diff
@@ -115,6 +115,13 @@
     name: "{{ satellite_package }}"
     state: latest
 
+# Install postgres server
+# this was not installed by satellite RPM in 6.4
+- name: Install postgresql-server
+  yum:
+    name: "postgresql-server"
+    state: latest
```

Comment:
Seems odd this wasn't installed by 6.4 rpm, did you ask the build team about it?

Reply:
Talked w/ eric about it. He mentioned that the dependency was removed due to the support of remote dbs so installation of the postgresql server is handled by the installer. I decided to install the RPM here because postgres user was missing. This raises the question of what we will do for cloning a 6.4 sat with a remote DB. What do you think?

Reply:
That makes sense, and ya, we do need the postgres user for certain tasks the way things currently are set up. 

I think a larger discussion about sat-clone and remote dbs is needed, but I imagine people would want support at some point. If we don't add it in this PR, we should update the docs to say cloning 6.4 installations with remote dbs isn't supported.